RecoLocalCalo/EcalRecAlgos/src/PulseChiSqSNNLS.cc

```diff
@@ -79,51 +79,82 @@ PulseChiSqSNNLS::PulseChiSqSNNLS() :
   _maxiterwarnings(true)
 {
   
-  Eigen::initParallel();
```

Comment:
Yes, this can be removed as per the discussion at https://hypernews.cern.ch/HyperNews/CMS/get/swDevelopment/3290/1/1/1/1/1/1/1/1.html and the change here https://github.com/cms-externals/eigen/pull/1

@Dr15Jones can comment further